css/components/style.css

```diff
@@ -18,3 +16,142 @@ body {
     font-weight: 300;
     font-style: normal;
 }
+
+#section-1 {
+  height: 670px;
+  position: relative;
+}
+
+.mw-background {
+  background: url('/assets/images/intro.jpg') no-repeat center center fixed;
+    background-size: 100%;
+    width:100%;
+    height: 100%;
+    opacity: 0.6;
+}
+
+.mw-header {
+  position: absolute;
+    top: 50%;
+    left: 15%;
+    font-size: 100px;
+}
+
+.mw-container-1 {
+  padding: 150px 200px 0;
+  height: 500px;
+  background-color: white;
+}
+
+.mw-container-1 > .mw-container-left {
+  height: 250px;
+  width: calc(50% - 20px);
+  display: inline-block;
+  background-image: url('/assets/images/section2.jpg') ;
+  background-size: 50%;
+  box-sizing: border-box;
+  background-size: cover;
+  background-position: center;
+  
+  
+}
+
+.mw-container-1 > .mw-container-right {
+  padding-left: 20px;
+  float: right;
+  height: 250px;
+  width: 50%;
+}
+
+.mw-container-1 > .mw-container-right > span:nth-child(1) {
+  display: block;
+  font-size: 70px;
+  padding-left: 20px;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-style: normal;
+}
+
+.mw-container-1 > .mw-container-right > span:nth-child(2) {
+  display: block;
+  padding-left: 30px;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-style: normal;
+}
+
+.mw-container-1 > .mw-container-right > button {
+  background-color: var(--color-2);
+  color: #fff;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  padding: 10px 20px;
+  margin: 20px 30px;
+}
+
+.mw-container-1 > .mw-container-bottom {
+  width: 100%;
+  height: 200px;
+  margin-top: 100px;
+}
+
+.mw-container-1 > .mw-container-bottom > .mw-box-1 > .mw-icon,
```

Comment:
poti in loc de .mw-box-1 , .mw-box-2 , .mw-box-3 , sa ai doar o singura clasa .mw-box, iar pe unde ai chestii mai specifice le selectezi cu :nth-child